src/coreclr/inc/readytorun.h

```diff
@@ -19,10 +19,10 @@
 //  src/coreclr/nativeaot/Runtime/inc/ModuleHeaders.h
 // If you update this, ensure you run `git grep MINIMUM_READYTORUN_MAJOR_VERSION`
 // and handle pending work.
-#define READYTORUN_MAJOR_VERSION 0x0009
-#define READYTORUN_MINOR_VERSION 0x0002
+#define READYTORUN_MAJOR_VERSION 0x000A
+#define READYTORUN_MINOR_VERSION 0x0000
```

Comment:
```suggestion
#define READYTORUN_MAJOR_VERSION 10
#define READYTORUN_MINOR_VERSION 0
```
We should switch to decimal for these version numbers. Version numbers in hex are unnatural.